nservicebus/msmq/subscription-persistence_config_core_[5,6].partial.md

```diff
@@ -0,0 +1,11 @@
+WARNING: Storing subscriptions in MSMQ is not suitable when endpoints are scaled-out, because the subscription queue cannot be shared among multiple endpoints.
```

Comment:
Should it be:

WARNING: Storing subscriptions in MSMQ is not suitable when endpoints are scaled-out, because the subscription queue cannot be shared among multiple endpoints, especially if the endpoints are running on different machines.

because they could run on the same machine in a competing consumer pattern on the same machine.